src/sandstorm/app-index/indexer.c++

```diff
@@ -248,7 +248,10 @@ void Indexer::updateIndex() {
 
       capnp::StreamFdMessageReader statusMessage(raiiOpen(statusFile, O_RDONLY));
       auto status = statusMessage.getRoot<SubmissionStatus>();
-      if (status.getRequestState() == SubmissionState::PUBLISH && status.isApproved()) {
+      if ((!unapprovedApps && status.getRequestState() == SubmissionState::PUBLISH &&
+            status.isApproved()) ||
+          (unapprovedApps && status.getRequestState() != SubmissionState::IGNORE &&
```

Comment:
This will reveal embargoed apps: SubmissionState::REVIEW means "review but don't publish yet".

I think you can change the whole if() statement to:

    if (status.getRequestState() == SubmissionState::PUBLISH &&
        status.isApproved() == !unapprovedApps)

This could be made clearer by reversing the meaning of the parameter: replace `bool unapprovedApps` with `bool approvedApps`. (Double-negatives are hard to reason about.)